src/bacnet/basic/service/h_apdu.c

```diff
@@ -511,6 +512,16 @@ static bool apdu_confirmed_dcc_disabled(uint8_t service_choice)
                 status = true;
                 break;
         }
+    } else if (dcc_communication_initiation_disabled()) {
```

Comment:
Please add `#if (BACNET_PROTOCOL_REVISION >= 20)` around these additions since they were added in protocol-revision 20.

Reply:
Added check for protocol-revision 20.